lib/cylc/scheduler_cli.py

```diff
@@ -140,26 +140,30 @@ def parse_commandline(is_restart):
         parser = COP(RESTART_DOC, jset=True, argdoc=[SUITE_NAME_ARG_DOC])
     else:
         parser = COP(
-            RUN_DOC, jset=True,
+            RUN_DOC,
+            icp=True,
+            jset=True,
             argdoc=[SUITE_NAME_ARG_DOC, START_POINT_ARG_DOC])
 
     parser.add_option(
-        "--non-daemon", help="(deprecated: use --no-detach)",
-        action="store_true", default=False, dest="no_detach")
+        "-n", "--no-detach", "--non-daemon", help="Do not daemonize the suite",
+        action="store_true", dest="no_detach")
 
     parser.add_option(
-        "-n", "--no-detach", help="Do not daemonize the suite",
-        action="store_true", default=False, dest="no_detach")
+        "-a", "--no-auto-shutdown", help="Do not shut down"
```

Comment:
This has been considered, but I think I'll stick with the current implementation for now as it seems to be closer to the Unix CLI norm.

I have considered these schemes:
* `--OPTION` `--no-OPTION` pair like in commands such as `rsync` and `gcc`. Chosen here as we already have a `--no-auto-shutdown` option.
* `-X` and -x` pair like in `ssh -X` and `ssh -x`.
* `-x` and `+x` pair. (I never like this one, because the positive option uses `-` whereas the negative option uses `+`.)
* `--option=yes|no` pair like in `ssh -oBatchMode=yes`.

Reply:
OK, for bool options a `no-` prefix isn't too bad.